[
  {
    "ClassName": "System.ApplicationException",
    "Message": "Throwing from inside of LogConsumerMock.",
    "Data": null,
    "HelpURL": null,
    "RemoteStackTraceString": null,
    "RemoteStackIndex": 0,
    "ExceptionMethod": null,
    "HResult": -2146232832,
    "Source": "Newsgirl.Shared.Tests",
    "WatsonBuckets": null
  }
]